src/components/Header.tsx

```diff
@@ -1,7 +1,13 @@
 import React from "react";
 
-const Header = () => {
-  return <div>Header</div>;
+const Header = (props: { position: string; title: string }) => {
+  return (
+    <div
+      className={`w-1/8 flex flex-row border-b-4 border-t-4 border-white p-2 pl-[5%] pr-[5%] text-3xl text-white ${props.position === "left" ? "border-r-4" : "border-l-4"}`}
```

Comment:
Are you sure `w-1/8` is a class in tailwind? 
Hover over it in vscode (tailwind intellisense)